src/main/java/org/jpeek/XslReport.java

```diff
@@ -109,10 +110,11 @@
      * Ctor.
      * @param xml Skeleton
      * @param name Name of the metric
+     * @param calc Calculus
      */
-    XslReport(final XML xml, final String name) {
+    XslReport(final XML xml, final String name, final Calculus calc) {
```

Comment:
@HDouss So, `XslReport` will be able to accept any other implementations of `Calculus` right?

Reply:
@fanifieiev yes!